docusaurus/docs/deployment.md

```diff
@@ -438,9 +438,9 @@ To support `pushState`, make sure to create a `public/_redirects` file with the
 
 When you build the project, Create React App will place the `public` folder contents into the build output.
 
-## [ZEIT Now](https://zeit.co)
+## [Vercel](https://vercel.com)
 
-[ZEIT Now](https://zeit.co) is a cloud platform for websites and serverless APIs, that you can use to deploy your Create React App projects to your personal domain (or a free `.now.sh` suffixed URL).
+[Vercel](https://vercel.com) is a cloud platform for websites and serverless APIs, that you can use to deploy your Create React App projects to your personal domain (or a free `.now.sh` suffixed URL).

```

Comment:
I think that the suffixes are now `vercel.sh`, right?

Reply:
@petetnt The default suffixes will change to `.vercel.app`, but they haven't been changed as of yet. It's still `.now.sh` for now.

I'll make another PR when it's updated to `.vercel.app`!